srcDoc/examples.rst

```diff
@@ -600,3 +600,359 @@ Maximum Throughput -- 2 *Routing Service*
+            success = false;
+        }
+        retcode = test_seq_data.set_long_seq(
+                    "test_seq",
+                    DDS_DYNAMIC_DATA_MEMBER_ID_UNSPECIFIED,
+                    long_seq);
+        if (retcode != DDS_RETCODE_OK) {
+            fprintf(stderr, "set_long(test_seq) failed: %d.\n", retcode);
+            success = false;
+        }
+        retcode = custom_type_data.unbind_complex_member(test_seq_data);
+        if (retcode != DDS_RETCODE_OK) {
+            fprintf(
+                    stderr,
+                    "bind_complex_member(test_seq_data) failed: %d.\n",
+                    retcode);
+        }
+        retcode = data.unbind_complex_member(custom_type_data);
+        if (retcode != DDS_RETCODE_OK) {
+            fprintf(
+                    stderr,
+                    "bind_complex_member(custom_type) failed: %d.\n",
+                    retcode);
+            success = false;
+        }
+        return success;
+    }
+
+- **finalize_custom_type_dynamic_data**:
+    This function is used to remove your data. It is called in the destructor.
+    The function takes one argument:
+
+        - A reference to the full DDS_DynamicData object including custom_type.
+
+::
+
+    bool finalize_custom_type_dynamic_data(DDS_DynamicData & data)
+    {
+        bool success = long_seq.ensure_length(0, 0);
+        if (!success) {
+            fprintf(stderr, "long_seq.ensure_length failed.\n");
+        }
+        DDS_ReturnCode_t retcode = data.clear_all_members();
+        if (retcode != DDS_RETCODE_OK) {
+            fprintf(stderr, "clear_all_members failed: %d.\n", retcode);
+            success = false;
+        }
+        return success;
+    }
+
+3. Build *RTI Perftest* using ``--customType <type>``
+
+::
+
+    ./build.sh --platform x64Linux3gcc5.4.0 --nddshome /home/rti_connext_dds-5.3.0 --cpp-build --customType Test
+
+4. Launching *RTI Perftest*
+
+::
+
+    ~/rtiperftest$ ./bin/x64Linux3gcc5.4.0/release/perftest_cpp -pub -executionTime 60 -noprint
+    Using Unkeyed Data.
+    Transport Information:
+        Kind: Default (UDPv4) / Custom (Taken from QoS profile)
+    Waiting to discover 1 subscribers...
+    Waiting for subscribers announcement ...
+    Publishing data...
+    Setting timeout to 60 seconds
+    Length:   463  Latency: Ave    266 us  Std   70.3 us  Min     28 us  Max    777 us  50%    260 us  90%    341 us  99%    465 us  99.99%    777 us  99.9999%    777 us 
+    Finishing test due to timer...
+    Test ended.
+
+
+::
+
+    ~/rtiperftest$ ./bin/x64Linux3gcc5.4.0/release/perftest_cpp -sub -noprint
+    Using Unkeyed Data.
+    Transport Information:
+        Kind: Default (UDPv4) / Custom (Taken from QoS profile)
+    Waiting to discover 1 publishers ...
+    Waiting for data...
+    Length:   463  Packets:  4990501  Packets/s(ave):   83175  Mbps(ave):   308.1  Lost: 0
+    Finishing test...
+    Test ended.
+
+
+4. Launching *RTI Perftest* with DynamicData
```

Comment:
This should be step 5.
Launch *RTI Perftest* with DynamicData.
(not Launching ... put period at end)